### ISSUE TITLE
Bug report : 

### ISSUE THREAD

**Issue by thowtue** [MommyHeather/AdvancedBackups#76]
**Describe the bug**
Compression lvl seems not to compress more at lvl 9 than lvl 4 or 1

**To Reproduce**
I tried different ratio compressions settings (in: config.advancedbackups.zips.compression) and my fullbackup .zip file is around the same size +- 2mb on around 700mb for all backups made.

The config.advancedbackups.chains.compress is set to _true_ as well so i don't understand what is going on.

**Expected behavior**
Should compress more the .zip files at lvl9 than lvl1

**Screenshots**
![rty](https://github.com/MommyHeather/AdvancedBackups/assets/34699555/5dac9cf9-b0cb-42cc-8b29-67bf4e710d4c)
Top one is lvl9 bottom one is lvl1

**Versioning :**
 - OS: Win 10 64bit 16GB ram 
 - Minecraft Version(s): 1.20.1
 - Modloader(s): Fabric
 - Mod Version: 3.5.3

**Additional context**
- launch args : -Xms8G -Xmx12G -Xmn12G
- 70mods loaded

**Comment by Jadan1213** [comment]
I would assume this is due to the compressibility of the files. There's generally diminishing returns the higher you go with compression, and the files can only compress so much.

**Comment by MommyHeather** [comment]
Yeah, things are pretty much as jadan said - compression is only so effective, and the compression level affects the *goal*, it just can't always be acheived.

It's also worth noting that the compression level in the config is (correctly) being passed to java's builtin `ZipOutputStream`, which handles the actual act of compressing - which does mean that if there *is* an issue, it's not really one I could fix as it'd be an issue in your build of java.

**Comment by thowtue** [comment]
Allright thank you

**Comment by Jadan1213** [comment]
> Allright thank you

My recommendation, to avoid wasting CPU cycles, would be to leave it at 1. You could try 2 or 3 if you really want, but it doesn't appear that it will make very much difference.

**Comment by thowtue** [comment]
Since changing the value doesn't seem to change anything i've put back the default value of 4. 
I don't think it changes anything with CPU use since it takes same amount of time and no more resources from the CPU when compressing at lv1 than lvl9
